stacks/vpc.yml

```diff
@@ -289,7 +326,20 @@ Resources:
       Protocol: 6 # TCP
       RuleAction : allow
       RuleNumber : 100
-  InboundHTTPNetworkAclEntry:
```

Comment:
So you renamed all these ACL entries - what will that do, exactly?  Ideally the new ones will be created before the old ones drop, so we're not missing traffic.

Reply:
Yes, that's the CloudFormation way